app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  # before_action :configure_sign_up_params, only: [:create]
```

Comment:
Should we delete commented code? It is git best practice

Reply:
yes, i know, but a huge part of code was generated with devise gem and this important for me in future